tools/istio-iptables/pkg/capture/run.go

```diff
@@ -616,7 +616,7 @@ func HandleDNSUDP(
 		ext:      ext,
 		ops:      ops,
 		table:    constants.NAT,
-		chain:    constants.OUTPUT,
```

Comment:
You need to insert the jump to the istio chain, since we are no longer using the default one.

Reply:
Thank you, @bleggett! I've added the required jumps in the latest commit. The integration tests look OK now.